composer.json

```diff
@@ -36,16 +33,12 @@
         "symfony/messenger": "^5.0",
         "symfony/monolog-bundle": "^3.0",
         "symfony/process": "^5.0",
-        "symfony/property-access": "^5.0",
-        "symfony/property-info": "^5.0",
         "symfony/proxy-manager-bridge": "^5.0",
         "symfony/security-bundle": "^5.0",
         "symfony/serializer": "^5.0",
-        "symfony/stopwatch": "^5.0",
         "symfony/twig-bundle": "^5.2",
```

Comment:
Does the twig bundle be part of direct dependencies or `dev` as well?

Sorry, not sure how this should work

Reply:
Twig is loaded indirectly through the profiler bundle in `dev` mode, so it will work :)